src/App.js

```diff
@@ -1,11 +1,62 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import Pokemon from "./Components/pokemon/Pokemon";
+import Button from "./Components/button/Button.js";
+
+
+
 
 function App() {
+
+    const [allPokemonData,setAllPokemonData] = useState([]);
```

Comment:
Als `pokemon` een object zal worden, dan zou ik een default waarde van `undefined` pakken. Nu lijkt het alsof het een array is en zal blijven, maar dat is niet zo
```suggestion
    const [allPokemonData,setAllPokemonData] = useState();
```